app/views/splash/index.html.erb

```diff
@@ -5,7 +5,11 @@
     <h1><%= application_name %></h1>
     <p><%= t('application.tagline') %></p>
     <p>
-      <%= link_to t('hyku.splash.account_signup'), account_sign_up_path, class: 'btn btn-lg btn-sign-up' %>
+      <% if !Settings.multitenancy.admin_only_tenant_creation || can?(:manage, Account) %>
```

Comment:
OK, so if admin-only tenant creation is OFF, or if it's ON and I am an admin, then I see the "Get Started" button?

Reply:
Correct. Essentially the "Get Started" button shows if you have permissions to create tenants (and in each of those scenarios you have permissions to create tenants). I could enhance specs to make this clearer, as needed.

Reply:
I've updated the PR description above to better describe how this functions. I now realize I rushed the PR description a bit on Friday as I was heading out the door ;)

Reply:
Just pushed up an enhanced `index.html.erb_spec.rb` which better details out expectations in various scenarios: https://github.com/samvera-labs/hyku/pull/1425/commits/aef2afd8d113d15ea718327361a9b5570cf5c89b#diff-351b2f70fd5d2ec8d5ea0fbe2bf8d3f3